client/src/pages/profile/UploadButton/UploadButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { useRef } from "react";
+import { Button } from "semantic-ui-react";
+
+import styles from "./UploadButton.module.css";
+
+const UploadButton = () => {
+  const fileInputRef = useRef<HTMLInputElement>(null);
+
+  const handleButtonClick = () => {
+    if (fileInputRef.current) {
+      fileInputRef.current.click();
+    }
+  };
+
+  //   const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
remove

Reply:
it will be needed for future functionalities